Packs/Code42/ReleaseNotes/5_0_0.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### New: Code42 Event Collector
+
+- New: Code42 Insider Risk software solutions provide the right balance of transparency, technology and training to detect and appropriately respond to data risk. Use the Code42EventCollector integration to fetch file events and audit logs.<~XSIAM> (Available from Cortex XSIAM 8.5.0).</~XSIAM>
```

Comment:
```suggestion
New: Code42 Insider Risk software solutions provide the right balance of transparency, technology and training to detect and appropriately respond to data risk. Use the Code42EventCollector integration to fetch file events and audit logs.<~XSIAM> (Available from Cortex XSIAM 8.5.0).</~XSIAM>
```
I don't think that is the correct version number for Cortex XSIAM. Do you mean 2.1?

Reply:
yes